src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -18,27 +19,62 @@
 
 public class DriveBaseSubsystem extends SubsystemBase {
   /**
-   * Creates a new ExampleSubsystem.
+   * Creates a new DriveBaseSubsystem.
```

Comment:
This comment should add more detail to what DriveBaseSubsystem does/is.
Ideally, it would be a description of the drive base it is meant to control and what devices, sensors, etc it uses. Other features like drive mode support and brief descriptions of the joystick mappings for each mode. 
Imagine next year coming back to the code and trying to remember what you did, and write down here what you think you might forget about how it is mean to work.